modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalAddressInput.js

```diff
@@ -342,6 +386,79 @@ const InfoBoxModalAddressInput = ({
 						/>
 					</ClayForm.Group>
 
+					{(additionalProps.addressSubtypeConfiguration.billing ||
+						additionalProps.addressSubtypeConfiguration
+							.billingAndShipping ||
+						additionalProps.addressSubtypeConfiguration
+							.shipping) && (
```

Comment:
We have `&&` in many other spots, but this is discouraged and a ternary returning a `null` is preferable to avoid unexpected value renders.